src/server/ast/operations.ts

```diff
@@ -33,7 +37,7 @@ export class ASTFunctionCall extends ASTNodeData implements ASTNode
 	}
 }
 
-class ASTUnaryOp extends ASTNodeData implements ASTNode
+export class ASTUnaryOp extends ASTNodeData implements ASTNode
```

Comment:
Those classes need to be visible in `test/testNodes.ts` so we can extend them with the `isEqualNode` test method

Reply:
Ok, we'd wondered if that might be the case. Could you please add a comment to the front of both mentioning that this is the only reason they're `export` and that they should not be used otherwise outside of the .ts